builtin/app/go/data/common/dev/Vagrantfile.tpl

```diff
@@ -8,6 +8,9 @@
   # Setup a synced folder from our working directory to /vagrant
   config.vm.synced_folder '{{ path.working }}', "{{ shared_folder_path }}",
     owner: "vagrant", group: "vagrant"
+  config.vm.provision "shell", inline:
+    "fstype=$(find /opt/gopath -mindepth 0 -maxdepth 0 -type d -printf '%F')
+    find /opt/gopath -fstype ${fstype} -print0 | sudo xargs -0 -n 100 chown vagrant:vagrant"
```

Comment:
You should be able to use `-mount` here instead of `-fstype ${fstype}`. It eliminates the need to run `find` twice.